parser/src/main/java/io/moderne/jenkins/ingest/Merger.java

```diff
@@ -0,0 +1,78 @@
+package io.moderne.jenkins.ingest;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Merger {
+
+    public static void main(String[] args) throws IOException {
```

Comment:
What do you think about including a test when this file is modified?

Reply:
Added just now.